cmd/puppeth/module_explorer.go

```diff
@@ -30,107 +30,80 @@ import (
 
 // explorerDockerfile is the Dockerfile required to run a block explorer.
 var explorerDockerfile = `
-FROM puppeth/explorer:latest
-
-ADD ethstats.json /ethstats.json
-ADD chain.json /chain.json
+FROM puppeth/blockscout:latest
 
+ADD genesis.json /genesis.json
 RUN \
-  echo '(cd ../eth-net-intelligence-api && pm2 start /ethstats.json)' >  explorer.sh && \
-	echo '(cd ../etherchain-light && npm start &)'                      >> explorer.sh && \
-	echo 'exec /parity/parity --chain=/chain.json --port={{.NodePort}} --tracing=on --fat-db=on --pruning=archive' >> explorer.sh
+  echo 'geth --cache 512 init /genesis.json' > explorer.sh && \
+	echo $'exec geth --networkid {{.NetworkID}} --syncmode "full"  --port {{.NodePort}} --bootnodes {{.Bootnodes}} --ethstats \'{{.Ethstats}}\' --cache=512 --rpc --rpccorsdomain "*" --rpcvhosts "*" --ws --wsorigins "*" &' >> explorer.sh && \
```

Comment:
I prefer to allow user to specify which geth node they want to connect to other than always using localhost

Reply:
My original idea is: this embedded geth node is only available for blockscout. Probably if a user wants to connect to a geth node, then they can deploy a new node somehow.

Reply:
Yes, it's a good idea. by the way, when I running embedded geth node the docker container suddenly shutdown. I might be wrong somewhere but still dont know the reason but when connect to outside node, it worked properly without any problem.